### ISSUE TITLE
ceph luminous rbd-provisioner flag redefined

### ISSUE THREAD

**Issue by SkinyMonkey** [kubernetes-incubator/external-storage#1150]
I tried to use the official provisionner image but it's configured to use mimic.
I built a luminous image but i get : 

```
/usr/local/bin/rbd-provisioner flag redefined: log_dir
panic: /usr/local/bin/rbd-provisioner flag redefined: log_dir

goroutine 1 [running]:
flag.(*FlagSet).Var(0xc0000b8120, 0x156f000, 0x215bed0, 0x138d8eb, 0x7, 0x13b6b7b, 0x2f)
        /usr/local/go/src/flag/flag.go:850 +0x4af
flag.(*FlagSet).StringVar(...)
        /usr/local/go/src/flag/flag.go:753
github.com/kubernetes-incubator/external-storage/vendor/k8s.io/klog.InitFlags(0x0)
        /go/src/github.com/kubernetes-incubator/external-storage/vendor/k8s.io/klog/klog.go:411 +0xa4
main.main()
        /go/src/github.com/kubernetes-incubator/external-storage/ceph/rbd/cmd/rbd-provisioner/main.go:46 +0x3f
```

After trying to fix the bug described in https://github.com/kubernetes-incubator/external-storage/issues/1120 with the solution that was merged in https://github.com/kubernetes-sigs/sig-storage-lib-external-provisioner/commit/99ac517d9c047238af8b8a6c411084b9f7888a87

**Comment by wongma7** [comment]
OK, if I understand correctly we need to update our dependency on sig-storage-lib-external-provisioner.
/assign

**Comment by lxs137** [comment]
meet same bug

**Comment by Alan-buaa** [comment]
cephfs have same bug, how to solve

**Comment by lxs137** [comment]
I try to fix this by https://github.com/kubernetes-sigs/sig-storage-lib-external-provisioner/pull/25, but still not works

**Comment by Alan-buaa** [comment]
I fix by https://github.com/kubernetes-incubator/external-storage/issues/1120#issuecomment-460684760

**Comment by lucheng001** [comment]
> I fix by [#1120 (comment)](https://github.com/kubernetes-incubator/external-storage/issues/1120#issuecomment-460684760)

I fix by delete klog.InitFlags(nil) in ceph/rbd/cmd/rbd-provisioner/main.go
and it works

**Comment by lucheng001** [comment]
> I fix by [#1120 (comment)](https://github.com/kubernetes-incubator/external-storage/issues/1120#issuecomment-460684760)

can fix by deleteing klog.InitFlags(nil) in ceph/rbd/cmd/rbd-provisioner/main.go
